.gitignore

```diff
@@ -1,7 +1,6 @@
 __pycache__/
 .vscode
 build/*
-*.zip
```

Comment:
This line was here to keep zip's created by stage.sh from being added to the repo. Needs to stay in at least some form. What zip are you adding? I don't see a zip in the PR.

Reply:
Fixed.